networks/movement/movement-full-node/src/node/tasks/transaction_ingress.rs

```diff
@@ -66,13 +66,13 @@ impl Task {
 			{
 				Ok(transaction) => match transaction {
 					Some((application_priority, transaction)) => {
-						info!(
+						tracing::debug!(
```

Comment:
Good, less log spam by default.
Could also import `debug` at the top, like `tracing::info`.